app/controllers/rooms_controller.rb

```diff
@@ -240,7 +242,16 @@ def validate_accepted_terms
 
   def validate_verified_email
     if current_user
-      redirect_to resend_path unless current_user.email_verified
+      redirect_to account_activation_path(current_user) unless current_user.email_verified
+    else
+      redirect_to root_path
+    end
+  end
+
+  def verify_room_owner_verified
+    unless @room.owner.email_verified
+      flash[:alert] = t("room.unavailable")
+      redirect_to root_path
```

Comment:
Same here, why to send the user to root?

Reply:
Would it be preferred to send the user to their main room with the current alert?